src/accelerate/utils/deepspeed.py

```diff
@@ -21,6 +21,10 @@
 from ..scheduler import AcceleratedScheduler
 
 
+def get_active_deepspeed_plugin(state):
+    return next((plugin for plugin in state.deepspeed_plugin if plugin.enabled), None)
```

Comment:
document that it'll return `None` if none are active?

But is this even possible? There must be at least one plugin active if ds has been configured, so if this plugin is called when there none active shouldn't it assert instead since it'd point to a user trying to call deepspeed w/o configuring it?

otherwise what is a user to do with `None`?